content/en/_index.md

```diff
@@ -104,19 +121,37 @@ to interact with a returned object programmatically by printing some example
 attributes to the console.
 
 {{< tabpane langEqualsHeader=true >}}
- {{< tab header="Java" >}}
+  {{< tab header="Java" >}}
     public static void main(String[] args) {
         // TODO(developer): Replace these variables before running the sample.
         String projectId = "my-project-id";
         String filePath = "path/to/image.png";
-        inspectImageFile(projectId, filePath);
+        exampleSnippet(projectId, filePath);
     }
 
     // This is an example snippet for showing best practices.
     public static void exampleSnippet(String projectId, String filePath) {
         // Snippet content ...
     }
-{{< /tab >}}
+  {{< /tab >}}
+  {{< tab header="Node.js" >}}
+    // Required values are passed as arguments to the main() function.
+    // Example declarations are included (but commented out) in the snippet.
+    function main(projectId, filePath) {
+      // [START example_snippet]
+      // TODO(developer): uncomment these variables and replace their values.
+      // const projectId = 'my-project';
+      // const filepath = 'path/to/image.png';
+      function exampleSnippet() {
+        // Snippet content...
+      }
+      // [END example_snippet]
+      exampleSnippet()
+    }
+
+    // Run the main function with command-line arguments
+    main(...process.argv.slice(2));
```

Comment:
I don't think we should take in arguements. That goes against the "No CLIs" idea.

Reply:
Reorganized this - the `main` method only invokes snippets with **default** arguments now.